forge-gui/res/cardsfolder/upcoming/flare_of_duplication.txt

```diff
@@ -0,0 +1,6 @@
+Name:Flare of Duplication
+ManaCost:1 R R
+Types:Instant
+S:Mode$ Alternative Cost | ValidSA$ Spell.Self | EffectZone$ All | Cost$ Sac<1/Creature.Red+nonToken/a non token red creature> | Description$ You may sacrifice a nontoken red creature rather than pay this spell's mana cost.
+A:SP$ CopySpellAbility | ValidTgts$ Instant, Sorcery | TargetType$ Spell | MayChooseTarget$ True | SpellDescription$ Copy target instant or sorcery spell. You may choose new targets for the copy.
+Oracle:You may sacrifice a nontoken red creature rather than pay this spell’s mana cost. Copy target instant or sorcery spell. You may choose new targets for the copy.
```

Comment:
spell’s > spell's

We can't use curlys in scripts (this goes for things like ' and " etc.)

Reply:
Good to know, just changed it! Thanks!

Reply:
I've unresolved this because it's the other way around :P

Lines 4 & 6 both need spell's (you've used the wrong one)